chapters/en/Unit 5 - Generative Models/Diffusion models/Introduction - Diffusions.md

```diff
@@ -0,0 +1,82 @@
+# Introduction to Diffusion Models
+
+What you will learn from this chapter:
+
+    - What are Diffusion models and how do they differ from GANs
+    - Major sub categories of Diffusion models
+    - Use cases of Diffusion models
+    - Drawback in Diffusion models
+
+
+## What are Diffusion models and how do they differ from GANs
+
+Diffusion models are a new and exciting area in computer vision that has shown impressive results in creating images. These generative models work on two stages, a forward diffusion stage and a reverse diffusion stage: first, they slightly change the input data by adding some noise, and then they try to undo these changes to get back to the original data. This process of making changes and then undoing them helps generate realistic images.
+
+These generative models raised the bar to a new level in the area
+of generative modeling, particularly referring to models such
+as Imagen and Latent Diffusion Models (LDMs). For instance consider the below images generated via such models
+
+![Example images generated using diffusion models](files/diffusion-eg.png)
+
+GANs were considered by many as state-of-the-art generative models in terms of the quality of the generated samples, before the recent rise of diffusion models. GANs are also known
+as being difficult to train due to their adversarial objective,
+and often suffer from mode collapse ( where the generator fails to capture the entire distribution of the training data, resulting in repetitive or limited samples). In contrast, diffusion models have a stable training process and provide more diversity
+because they are likelihood-based.
```

Comment:
Can we add here 1 more line on about that : GANs producing high-fidelity, low-diversity samples. Diffusion is on he other hand? WDYT?